mlir/lib/Catalyst/Transforms/DetectQNodes.cpp

```diff
@@ -284,29 +279,35 @@ void DetectQnodeTransform::rewrite(LLVM::CallOp callOp, PatternRewriter &rewrite
         PatternRewriter::InsertionGuard insertGuard(rewriter);
         rewriter.setInsertionPointToEnd(failBlock);
         rewriter.create<LLVM::UnreachableOp>(callOp->getLoc());
-        return;
```

Comment:
The deletion of this "return" should be placed in another commit, as it is not related to "Documentation".